src/EA.Weee.RequestHandlers/AatfReturn/ReturnSchemeDataAccess.cs

```diff
@@ -39,11 +39,11 @@ public async Task<Organisation> GetOrganisationByReturnId(Guid returnId)
             return @return?.Organisation;
         }
 
-        public async Task RemoveReturnScheme(List<Guid> schemeIds)
+        public async Task RemoveReturnScheme(List<Guid> schemeIds, Guid returnId)
```

Comment:
Can we write an integration test for this method to ensure its only the returns we won't data deleted